js/index.js

```diff
@@ -1,6 +1,76 @@
+console.log("I'm ready!");
 // Iteration 1: Names and Input
+let hacker1="Alex";
+let hacker2="Modest";
+
+
+console.log(`The driver's name is ${hacker1}`);
+console.log(`The navigator's name is ${hacker2}`);
 //
 // Iteration 2: Conditionals
 
+if (hacker1.length === hacker2.length){
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`);
+}
+else if (hacker1.length > hacker2.length){
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`);
+}
+else{
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`);
+}
 
 // Iteration 3: Loops
+
+let hacker1SeparatedLetters='';
+for (let i=0; i<=hacker1.length-1; i++){
+    hacker1SeparatedLetters += hacker1[i] + " ";
+}
+console.log(hacker1SeparatedLetters.toUpperCase());
+
+let hacker2Reverse='';
+for(let i=hacker2.length-1; i>=0; i--){
+    hacker2Reverse += hacker2[i];
+}
+console.log(hacker2Reverse);
+
+
+const alphabet="abcdefghijklmnopqrstuvwxyz";
```

Comment:
En este caso te has complicado demasiado, con la función locale.compare() nos permite saber que letra va en orden alfabético. 
`
let lexic = hacker1.localeCompare(hacker2)
console.log(lexic)

if (lexic === -1) {
  console.log(`The driver's name goes first.`)
}
else if (lexic === 1) {
  console.log(`Yo, the navigator goes first definitely.`)
}
else {console.log(`What?! You both have the same name?`)}
`